dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+| Legacy Liferay components (now in Clay)                   |          | ✅     | ✅      | ✅    |
+| Common Liferay components (Portlets)                      |          | ✅     | ✅      | ✅    |
+| Liferay customizations of Clay components                 |          | ✅     | ✅      | ✅    |
+| Compat Layer: compatibility layer for Bootstrap 2 and 3   |          | ✅     | ✅      | ✅    |
+| Main CSS of the Liferay websites and pages                |          |        | ✅      |       |
+| CSS Custom Properties Layer: Modern customization of Clay |          |        | ✅      |       |
+| Hello World Widget: Front Page Banner                     |          |        | ✅      |       |
+| Main CSS of the Liferay administration panels             |          |        |         | ✅    |
+
+From a more technical point of view, setting the base through the generator will create a setting named `liferayTheme` in the `package.json` file of the theme folder.
+
+```json
+{
+	"name": "my-liferay-theme",
+	"version": "1.0.0",
+	"main": "package.json",
+	"keywords": ["liferay-theme"],
+	"liferayTheme": {
+		"baseTheme": "styled",
+		"templateLanguage": "ftl",
+		"version": "7.0"
+	},
+	"devDependencies": {
+		"gulp": "^3.8.10",
+		"liferay-theme-deps-7.1": "*",
+		"liferay-theme-tasks": "*"
+	}
+}
+```
+
+[Source](https://github.com/liferay/liferay-js-themes-toolkit/tree/master/packages/liferay-theme-tasks#liferaytheme)
+
+## How to import Clay into a custom-theme?
+
+We always recommend to use [liferay-js-themes-toolkit](https://github.com/liferay/liferay-js-themes-toolkit) to create or update a theme, but we understand that sometimes you could have the need of adding Clay to an old heavy-customized theme that can't be automatically updated. In those cases we will have to import Clay as a depencency.
+
+The first step will be to localize the project's folder and install the right dependencies. You can either use the classic Node system or the modern Yarn one.
+
+Node
+`npm install @clayui/css`
+
+Yarn
+`yarn add @clayui/css`
+
+It's important to know that these commands will only install the CSS library, if you are building a theme that can make a good use of Clay components created in React, we recommend to use:
+
+Node
+`npm install @clayui/*`
+
+Yarn
+`yarn add @clayui/*`
+
+You can also address specific components with commands like `npm install @clayui/buttons`, `yarn add @clayui/cards`, `yarn add @clayui/tables@^version`, etc. [Read more at Clay's website](https://clayui.com/docs/get-started/how-to-use-clay.html)
+
+## How is Clay imported into a module?
+
+Unlike the previous section we won't need to install Clay as a dependency in a module inside Liferay Portal. Thanks to our rich environment, we already have it included and we just need to enable it.
+
+To do so, we need to define the new import into our dedicated buildCSS process, we can do that copy the following code into the `build.gradle` file of our module.
```

Comment:
> we can do that copy the following code

we can do that by copying the following code

Reply:
buildCSS -> `buildCSS`